app/models/manageiq/providers/embedded_terraform/automation_manager/job.rb

```diff
@@ -106,8 +124,8 @@ def stack_response
     @stack_response ||= Terraform::Runner::ResponseAsync.new(options[:terraform_stack_id])
   end
 
-  def decrypt_extra_vars(extra_vars)
-    result = extra_vars.deep_dup
+  def decrypt_input_vars(input_vars)
```

Comment:
Maybe just rename this `decrypt_vars` since it could work for extra_vars or input_vars?  It doesn't depend on anything other than the argument you pass in.